test/test_arguments.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+from StringIO import StringIO
+from argparse import Namespace
+
+import pytest
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
+from genestack_client import get_user, make_connection_parser
+from genestack_client.settings.genestack_user import User
+
+
+def test_parse_default_user():
+    user = get_user(make_connection_parser().parse_args())
+    expected = User('tester@genestack.com', alias='tester',
+                    host='localhost:8080', password='pwdTester123')
```

Comment:
These seem perfect candidates for being defined as constants.

Reply:
Lets **test** expected values will be as is.